executor/common_linux.h

```diff
@@ -257,8 +257,8 @@ static void netlink_device_change(int sock, const char* name, bool up,
 	memset(&hdr, 0, sizeof(hdr));
 	if (up)
 		hdr.ifi_flags = hdr.ifi_change = IFF_UP;
+	hdr.ifi_index = if_nametoindex(name);
```

Comment:
What does if_nametoindex do underneath?

Reply:
I see we already use it in multiple places, so I guess it does not matter much.

Reply:
It just gets ifindex by netdevice name. Ifindex is then used as a handle.